financialaid/migrations/0003_added_timestamps.py

```diff
@@ -4,7 +4,7 @@
 
 import datetime
 from django.db import migrations, models
-
+import pytz
```

Comment:
Understood.
BUT, do we need the extra migration anyway to convert the existing data?